src/views/Settings.vue

```diff
@@ -81,14 +81,14 @@
 
       <v-col cols="12" sm="6" xl="4">
         <v-card>  
-          <v-card-title>මෘදුකාංගය යාවත්කාලින (update) කිරීම</v-card-title> 
+          <v-card-title>මෘදුකාංගය යාවත්කාල කිරීම</v-card-title> 
           <v-card-text>
-            <div>{{ `ඔබ දැන් භාවිතා කරන්නේ version: ${version} වන මෘදුකාංගයයි.` }}</div>
+            <div>{{ `ඔබගේ වත්මන් අනුවාදය: ${version}` }}</div>
```

Comment:
Should include the (version) as well, most people do not know what anuwada means. Even I did not know

Reply:
එය හඳුනා ගැනීමට අපහසු නැහැ. `අනුවාදය (version)` යන්න මෘදුකාංග සහ යෙදුම්වල අනිවාර්යෙන් ඇස ගැටෙන අංගයක් නිසා ඔබගේ යෝජනාව ප්‍රතික්‍ෂේප කරමි. අපි මෙයට විසඳුමක් ලෙස සිංහල භාෂා සහාය සහිත මෘදුකාංග ජනතාවට වඩාත් සමීප කිරීමට උත්සාහ කරන්නෙමු. https://maduraonline.com/?find=version

Reply:
This is the first time I have seen the word anuvadaya used for version.

Reply:
වඩාත් සුදුසු වචනයක් ඇත්නම් යෝජනා කරන්න. අපට එය සලකා බැලීමට හැකිය.
_තොරතුරු: copy, edit / edition, duplicate, modify, revision, change ආදියට අනන්‍ය වචන පවත්වා ගත යුතුය._